core/java/iesi-core/src/main/java/io/metadew/iesi/script/action/sql/SqlExecuteQuery.java

```diff
@@ -117,9 +117,14 @@ protected boolean executeAction() throws SQLException, InterruptedException {
         String outputDatasetReferenceName = convertDatasetReferenceName(getSqlQuery().getValue());
         boolean appendOutput = convertAppendOutput(getConnectionName().getValue());
         // Get Connection
-        Connection connection = ConnectionConfiguration.getInstance()
-                .get(new ConnectionKey(connectionName, this.getExecutionControl().getEnvName()))
-                .get();
+        Connection connection;
+        try {
+            connection = ConnectionConfiguration.getInstance()
+                    .get(new ConnectionKey(connectionName, this.getExecutionControl().getEnvName()))
+                    .get();
+        } catch (Exception e) {
+            throw new RuntimeException("Unknown connection name");
+        }
```

Comment:
instead of a try-catch clause, replace .get() with orElseThrow(() -> new RuntimeException(...))

Reply:
Corrected in next commit